<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceArgSetForm</class>
 <widget class="QWidget" name="DeviceArgSetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>设备参数设置</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	background-color: rgb(50, 50, 50);
}

QComboBox {
	color: #ffffff;
	background-color: rgba(255, 255, 255, 0);
	padding: 0px 1px;
	border: 1px solid #ffffff;
	font: 9pt &quot;黑体&quot;;
}

QComboBox::drop-down {
    width: 16px;
    border-left: 0px solid #000000;
	background-color: rgba(255, 255, 255, 50);
}

QComboBox::down-arrow {
	width: 16px;
    height: 16px;
	image: url(:/image/arrow-down.svg);
}

QComboBox QAbstractItemView {
	background-color: #323232;
    border: 0px solid #000000;
    outline: 0px;
}

QComboBox QAbstractItemView::item {
	color: #ffffff;
	height: 18px;
    padding-left: 9px;
	border: 0px solid #000000;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #646464;
}

QComboBox:disabled {
	color:  #999999;
	border: 1px solid #999999; 
 }

QComboBox::down-arrow:disabled {
	image: url(:/image/disable_ico.svg);
 }

QPushButton {
	background-color: rgba(255, 255, 255, 0.08);
	color: rgba(255, 255, 255, 0.85);
	min-width: 45px;
	min-height: 22px;
	outline: none;
	border-radius: 6px; 
            
	/* 3D 效果 */
	border-top: 1px solid rgba(255, 255, 255, 0.15);
	border-left: 1px solid rgba(255, 255, 255, 0.15);
	border-bottom: 1px solid rgba(0, 0, 0, 0.3);
	border-right: 1px solid rgba(0, 0, 0, 0.3);
 }

QPushButton:hover {
	background-color: rgba(255, 255, 255, 0.12);
	color: rgba(255, 255, 255, 0.95);

	border-top: 1px solid rgba(255, 255, 255, 0.2);
	border-left: 1px solid rgba(255, 255, 255, 0.2);
	border-bottom: 1px solid rgba(0, 0, 0, 0.4);
	border-right: 1px solid rgba(0, 0, 0, 0.4);
 }

QPushButton:pressed {
	background-color: rgba(255, 255, 255, 0.05);
	color: rgba(255, 255, 255, 0.8);
            
	/* 反转边框实现凹陷效果 */
	border-top: 1px solid rgba(0, 0, 0, 0.3);
	border-left: 1px solid rgba(0, 0, 0, 0.3);
	border-bottom: 1px solid rgba(255, 255, 255, 0.15);
	border-right: 1px solid rgba(255, 255, 255, 0.15);
 }

QPushButton:disabled {
	background-color: rgba(255, 255, 255, 0.03);
	color: #999999;

	border-top: 1px solid rgba(255, 255, 255, 0.05);
	border-left: 1px solid rgba(255, 255, 255, 0.05);
	border-bottom: 1px solid rgba(0, 0, 0, 0.1);
	border-right: 1px solid rgba(0, 0, 0, 0.1);
 }

QSpinBox {
    background-color: rgba(255, 255, 255, 0); 
    color: #ffffff;
    border: 1px solid #ffffff;
	min-height: 18px;
}

QSpinBox::up-button, QSpinBox::down-button {
    width: 0px;
    margin: 0px;
}

QSpinBox::up-arrow, QSpinBox::down-arrow {
    image: none;
}

QSpinBox:disabled {
	color:  #999999;
	border: 1px solid #999999;
 }

QCheckBox {
	color: #ffffff;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
}

QCheckBox::indicator:checked {
	image: url(:/image/checkbox.svg);
}

QCheckBox::indicator:checked:disabled {
	image: url(:/image/checkbox_disale.svg);
 }

QCheckBox::indicator:unchecked {
	image: url(:/image/uncheckbox.svg);
}

QCheckBox::indicator:unchecked:disabled {
	image: url(:/image/uncheckbox_disable.svg);
 }

QLabel {
	color: rgb(255, 255, 255);
}

 QLabel:disabled {
	color: #999999;
	background-color: rgba(255, 255, 255, 0);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="passwordWidget" native="true">
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,2,1">
      <item row="3" column="1">
       <widget class="QLabel" name="info">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="NO">
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="OK">
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="password">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(200, 200, 200);</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::EchoMode::Password</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0" rowspan="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>请输入管理员密码：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="4">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="setWidget" native="true">
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,1" columnstretch="1,0,1">
      <item row="1" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="0">
         <widget class="QCheckBox" name="flyFilterB">
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="text">
           <string>飞点滤波</string>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>阈值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>曝光时间</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>帧率设置</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QSpinBox" name="timeFilterV">
          <property name="minimum">
           <number>2</number>
          </property>
          <property name="maximum">
           <number>28</number>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QCheckBox" name="confFilterB">
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="text">
           <string>置信度滤波</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>阈值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QSpinBox" name="confFilterV">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>150</number>
          </property>
         </widget>
        </item>
        <item row="1" column="2" colspan="2">
         <widget class="QSpinBox" name="exposureTime">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>4000</number>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="spatialFilterB">
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="text">
           <string>空间滤波</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>阈值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QComboBox" name="timeFilterV_">
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>9</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="2" column="2" colspan="2">
         <widget class="QSpinBox" name="frameRate">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>28</number>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QSpinBox" name="flyFilterV">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>64</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>阈值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="7" column="3">
         <widget class="QComboBox" name="spatialFilterV">
          <property name="minimumSize">
           <size>
            <width>50</width>
            <height>0</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>3</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>5</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>7</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QCheckBox" name="timeFilterB">
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="text">
           <string>时域均值滤波</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QCheckBox" name="timeFilterB_">
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="text">
           <string>时域中值滤波</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="minimumSize">
             <size>
              <width>47</width>
              <height>26</height>
             </size>
            </property>
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;注：输入框在输入完值后须按“回车键”才能设置生效；曝光时间的设置会影响点云的曝光等级（每40数值对应一个等级）；帧率设置会影响曝光时间的设置范围（值越大范围越小，最小会缩短至：0-1000），反之亦然。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	color: rgb(255, 255, 255);
	image: url(:/image/notice.svg);
	background-color: rgba(255, 255, 255, 0);
	border-color: rgba(255, 255, 255, 0);
}

QToolTip {
	color: #ffffff;
	background-color: rgba(255, 255, 255, 0);
	border: 0px solid #ffffff;
	font: bold 12px 'Arial';
}</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="4" column="2">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>阈值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="8" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QCheckBox" name="isAutoReboot">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <italic>false</italic>
              <bold>true</bold>
              <underline>false</underline>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(63, 147, 200);</string>
            </property>
            <property name="text">
             <string>断开连接设备自启动</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_8">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="9" column="0" colspan="4">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>15</number>
          </property>
          <item>
           <widget class="QPushButton" name="saveConfig">
            <property name="text">
             <string>保存到设备</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="quit">
            <property name="text">
             <string>退出</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1">
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
